src/main/webapp/app/exam/manage/exams/exam-detail.component.html

```diff
@@ -130,9 +130,9 @@ <h3>Exam Details:</h3>
                     <span *ngIf="exam.examStudentReviewEnd; else examStudentReviewEndNotSet">{{ exam.examStudentReviewEnd | artemisDate }}</span>
                     <ng-template #examStudentReviewEndNotSet><span jhiTranslate="artemisApp.exercise.dateNotSet"></span></ng-template>
                 </dd>
-                <dt><span jhiTranslate="artemisApp.examManagement.workingTime">Working time (minutes)</span></dt>
+                <dt><span jhiTranslate="artemisApp.examManagement.workingTime">Working Time</span></dt>
```

Comment:
You need to update the translation keys for both EN and DE as well to actually change this in the UI :)